packages/office-ui-fabric-react/src/components/Link/Link.scss

```diff
@@ -42,6 +41,11 @@
   &:active {
     color: $ms-color-themePrimary;
   }
+
+  &:focus {
```

Comment:
Done. I've split out several styles like this that only need to apply to buttons or anchors, not both.